ecosystem/theme-default/src/client/styles/vars.scss

```diff
@@ -29,16 +31,28 @@
   --c-tip-title: var(--c-text);
   --c-tip-text: var(--c-text);
   --c-tip-text-accent: var(--c-text-accent);
-  --c-warning: #e7c000;
+  --c-warning: #ffc310;
```

Comment:
There vars are also used by badge component. Could you help to check the styles of them?

Reply:
Done at commit https://github.com/vuepress/vuepress-next/pull/1131/commits/1bb9e38135802998805e919bd3a34504d6519339